build_openwebtext_pretraining_dataset.py

```diff
@@ -43,7 +43,8 @@ def log(*args):
       output_dir=os.path.join(args.data_dir, "pretrain_tfrecords"),
       max_seq_length=args.max_seq_length,
       num_jobs=args.num_processes,
-      blanks_separate_docs=False
+      blanks_separate_docs=False,
```

Comment:
The doc says True by default.

Reply:
By now it is only `False` for the  OpenWebTextCorpus script. The main readme mentions the more generic `build_pretraining_dataset.py` script, where it is set to `True` :)